src/web/src/components/Posts/Post.tsx

```diff
@@ -284,7 +298,7 @@ const PostComponent = ({ postUrl }: Props) => {
       <div className={classes.content}>
         <section
           ref={sectionEl}
-          className="telescope-post-content"
+          className={'telescope-post-content' + blogClassName}
```

Comment:
See my comment above.  I would call a function here that returns the string you want.